CHANGELOG.md

```diff
@@ -126,7 +126,7 @@ Features:
   This is now done via a TOML table, rather than a specifically patterned string.
   Users who have the old style configuration will now receive a
   `DeprecationWarning` stating that the old behavior will go away with v3.0.
-* Fixed up warning handling so that we can display `DeprecationWarning`s to users
+* Fixed up warning handling so that we can display `DeprecationWarnings` to users
```

Comment:
This is probably more correct the original way, as the `s` is intended to be plural rather than part of the type.